packages/vitest/src/runtime/runners/index.ts

```diff
@@ -39,6 +39,7 @@ export async function resolveTestRunner(config: ResolvedConfig, executor: Vitest
     throw new Error('Runner must implement "importFile" method.')
 
   testRunner.config.diffOptions = await loadDiffConfig(config, executor)
+  await loadSnapshotSerializers(config, executor)
```

Comment:
I think we can load it in parallel for better performance:

```ts
const [diffOptions] = await Promise.all([
  loadDiffConfig(config, executor),
  loadSnapshotSerializers(config, executor),
])
```

Reply:
Make sense!